creating_images/guidelines.adoc

```diff
@@ -234,11 +234,19 @@ See the following references for other guidelines:
 - Docker documentation - https://docs.docker.com/articles/dockerfile_best-practices/[Best practices for writing Dockerfiles]
 - Project Atomic documentation - http://www.projectatomic.io/docs/docker-image-author-guidance/[Guidance for Container Image Authors]
 
-
+[[openshift-specific-guidelines]]
 == {product-title}-Specific Guidelines
 The following are guidelines that apply when creating container images specifically
 for use on {product-title}.
-
+ifdef::openshift-online[]
+*Privileges and Volume Builds*
+
+Images cannot use the `VOLUME` directive. Images using a read/write file system
```

Comment:
I think we might want to be a bit more explicit here: `Docker images cannot be built using the 'VOLUME' directive.`

Reply:
Vote for this comment.